spec.html

```diff
@@ -27824,225 +27824,121 @@ <h1>Function Properties of the Math Object</h1>
       <emu-clause id="sec-math.abs">
         <h1>Math.abs ( _x_ )</h1>
         <p>Returns the absolute value of _x_; the result has the same magnitude as _x_ but has positive sign.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *+&infin;*.
-          </li>
-        </ul>
+        <emu-alg>
+            1. Let _n_ be ? ToNumber(_x_).
+            1. If Number.isNaN(_n_), return *NaN*.
```

Comment:
this can't observably call into `Number.isNaN`; I think you may mean:
```suggestion
            1. If _n_ is *NaN*, return *NaN*.
```